src/tools/std20/functional/test/identity.test.cpp

```diff
@@ -0,0 +1,16 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/functional.hpp"
+
+// other includes
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano::ranges;
+
+SCENARIO( "invoke" ) {
```

Comment:
"identity" ?

Reply:
Yes, a copy/paste error.

Reply:
I will put all updates following the reviews of these PRs to the fix/review branch.